_dropin-templates/dropin-installation.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: DROPIN_NAME installation
+description: Learn how to install the DROPIN_NAME drop-in component into your site.
+sidebar:
+  # label: DROPIN_NAME Installation
+  order: 2
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+import Callouts from '@components/Callouts.astro';
+import Diagram from '@components/Diagram.astro';
+import Vocabulary from '@components/Vocabulary.astro';
+
+The DROPIN_NAME drop-in component, like our other drop-in components, are designed for the browser's JavaScript runtime without the need for a bundler. You can also install and execute drop-in components in a build-time environment with bundlers like Vite. The installation steps for both runtime and build-time environments are the same after the initial drop-in component package imports.
+
+## Step-by-step
+
+Use the following steps to install the DROPIN_NAME component:
+
+:::note[Edge Delivery Service]
+These steps are specific for EDS projects, and may not necessarily be the same for other frameworks such as a static HTML or React framework.
+:::
+
+<Tasks>
+
+<Task>
+
+### Install the packages
+
+Use a CDN or NPM (recommended for performance) to install the drop-in component tools (`@dropins/tools`) and DROPIN_NAME (`@dropins/storefront-DROPIN_PACKAGE`) packages.
```

Comment:
We don't use CDN in our Commerce Boilerplate, nor do we recommend CDN for production. We should simplify and only include NPM example.